features/ios/ios_native_errors.feature

```diff
@@ -22,6 +22,7 @@ Feature: iOS Native Errors
     And I discard the oldest error
     And the exception "message" equals "Last Run Info Correct"
 
+  @skip_unity_2018
```

Comment:
What happened here?  How did changing the team Id make it necessary to skip 2 tests?

Reply:
Oh, I see now - that seems fine to skip it, just bear in mind the need to test the rest of the skipped scenario if we happen to change this area of functionality (seem unlikely).